flask_webapp/app.py

```diff
@@ -25,10 +34,18 @@ def scan():
 
     return jsonify(retDict)
 
+@app.route('/video_feed')
+def video_feed():
+    print('Turn on Webcam')
```

Comment:
we may need a turn off method, and this camera state will have to be stored in the db.

Reply:
Noted. I will look at that.

thanks.